R/register-chunk-hook.R

```diff
@@ -0,0 +1,47 @@
+#' Use vcr in vignettes
+#'
+#' @description
+#' `register_chunk_hook()` registers a `use_cassette` hook so that you map
+#' a vcr cassette to a knitr chunk. Call it in your setup chunk:
+#'
+#' ````
+#' ```{r setup}
+#' #| include: false
+#' vcr::register_chunk_hook()
+#' ```
+#' ````
+#'
+#' Then in a chunk you want to use a cassette, set the `cassette` chunk option
+#' to the name of the cassette:
+#'
+#' ````
+#' ```{r}
+#' #| cassette: name
+#' req <- httr2::request("http://r-project.org")
+#' resp <- httr2::req_perform(req)
+#' ```
+#' ````
+#'
+#' @param prefix An optional prefix for the cassette name so that you only
+#'   need to make sure that casssette names are unique within the current
```

Comment:
```suggestion
#'   need to make sure that cassette names are unique within the current
```

I suggest going through all vcr docs to find occurrences of cassette written with the wrong number of s (last time I looked for `casette`) :joy_cat: